### ISSUE TITLE
PSX Emu crashing reaching end of disc (Audio tracks).

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TASVideos/BizHawk#372]
```
What steps will reproduce the problem?
1.Pick any game with audio tracks and a .cue file
2.run disc, Open virtual psx lid and run cd audio player, and close the lid.
3.Jump to final track and fast forward to the end.

What is the expected output? What do you see instead?
When reaching the end of the last track the emu will close down with the 
attached error.  This can easily happen playing the game as well

What version of the product are you using? On what operating system?
SVN8832.  windows 8 64bit. 

Please provide any additional information below.
The 2 second offset was fixed and tracks line up correctly.

When using .ccd files, the 2 second offset still exists and also crashes the 
emu at the last 2 seconds of the final track.

A similar issue used to exist for Sega CD as well, but has since been fixed.

```

Original issue reported on code.google.com by `MThompso...@gmail.com` on 18 Jan 2015 at 7:37

Attachments:
* [error.png](https://storage.googleapis.com/google-code-attachments/bizhawk/issue-372/comment-0/error.png)

**Comment by zeromus** [comment]
Presently, it's being caused by "lack of lead-out track synthesis"

**Comment by zeromus** [comment]
fixed with commit af8842c24f53bb9628e26ffedf050bc617fb9c05
